### PR TITLE
Calibre-Web prereqs for audio/video + set app.db logging to /var/log/calibre-web.log w/ log level "DEBUG"

### PR DESCRIPTION
**Pull request by holta**
For @deldesir's ongoing efforts to make Calibre-Web more useful to IIAB communities.

### CODE REVIEW

**Comment by holta** [comment]
FYI this was tested on the latest pre-release of Ubuntu Server 24.04

It's a WIP (work-in-progress) for sure, but is pointing in the right direction and should probably be merged ASAP to allow much more automated and/or meaningful testing going forward.

File: roles/calibre-web/files/app.db
Comment (deldesir) [inline roles/calibre-web/files/app.db:1]: Good, I noticed the config_logfile field is set to /var/log/calibre-web.log.

File: roles/calibre-web/tasks/install.yml
Comment (deldesir) [inline roles/calibre-web/tasks/install.yml:1]: Looks good to me.

**Review by deldesir — APPROVED** [review]
Well done.

**Pull request merged**